elixir/apps/web/lib/web/live/settings/api_clients/new.ex

```diff
@@ -4,18 +4,19 @@ defmodule Web.Settings.ApiClients.New do
   alias Domain.Actors
 
   def mount(_params, _session, socket) do
-    unless Domain.Config.global_feature_enabled?(:rest_api),
-      do: raise(Web.LiveErrors.NotFoundError)
+    if Domain.Accounts.rest_api_enabled?(socket.assigns.account) do
+      changeset = Actors.new_actor(%{type: :api_client})
 
-    changeset = Actors.new_actor(%{type: :api_client})
-
-    socket =
-      assign(socket,
-        form: to_form(changeset),
-        page_title: "Add an API Client"
-      )
+      socket =
+        assign(socket,
+          form: to_form(changeset),
+          page_title: "Add an API Client"
```

Comment:
```suggestion
          page_title: "New API Client"
```

(nit) to align with convention from #5938

Reply:
I'm not sure I'm following.  In #5938 it looks like you changed it from "New API Client" -> "Add an API Client".  Am I seeing that correctly?

Reply:
Whoops! I meant to make all of those page titles consistent. I can do that real quick.

Reply:
Ah, ok, so you do want all of the pages to have `New ...` ?

Reply:
Yeah not a big deal either way, just figured the `Add` terminology is already used in the breadcrumbs and button.

Or maybe it's better to be consistent. Can go either way.

Reply:
Yeah, no preference here, just wanted to make sure I understood and made the right change.  Your last PR made it clear.

Reply:
👍